modules/retail/pages/retail-deploy-terminals.adoc

```diff
@@ -0,0 +1,84 @@
+
+[IMPORTANT]
+====
+After you have registered new terminals, always check the {susemgr} {webui} to ensure your terminal has connected successfully to the branch server, and you have not connected directly to the {susemgr} server by mistake.
+====
+
+
+== Create a Hardware Type group
+
+Each terminal requires a specific hardware type, which contains information about the product name and terminal manufacturer.
+However, the {susemgr} database does not have this information to begin with.
+In order to tell {susemgr} what image to deploy on each terminal, you can set hardware type groups.
+After you have created your new hardware type group, you can apply the [systemitem]``saltboot-formula`` to the group and configure it for your environment.
+
+Hardware types allow you to group devices according to manufacturer and device name, so that all devices of a particular type can be managed as one.
+
+You will need to create appropriate hardware types in your system.
+
+Empty profiles can be assigned to a hardware type group either before or after registration.
+If an empty profile is not assigned to a hardware type group before registration, it will be assigned to group that best matches the product information available to it.
+
+For this procedure, you will require the system manufacturer name and product name for your terminal.
+
+.Procedure: Creating a Hardware Type group
+
+. Determine the hardware type group name.
+Prefix the name with [systemitem]``HWTYPE:``, followed by the system manufacturer name and product name, separated by a hyphen.
+For example:
++
+----
+HWTYPE:POSVendor-Terminal1
+----
++
+[IMPORTANT]
+====
+Only use colons (``:``), hyphens (``-``) or underscores (``_``) in hardware type group names.
+Spaces and other non-alphanumeric characters will be removed when the name is processed.
+====
++
+. In the {susemgr} {webui}, navigate to menu:Systems[System Groups], and click the btn:[Create Group] button.
+. In the [guimenu]``Create System Group`` dialog, create a new system group, using the hardware type group name you created earlier.
+
+
+== Assign and Configure the Saltboot formula for each Hardware Type Group
+
+TBD
+
+.Procedure: Configuring partitioning and assigned images
+
+. Open the details page for your new hardware type group, and navigate to the [guimenu]``Formulas`` tab.
+. Select the [systemitem]``saltboot-formula`` and click btn:[Save].
+. Navigate to the new menu:Formulas[Saltboot] tab.
+. Configure the Saltboot formula according to the instructions in xref:retail:retail-formulas[].
+. Apply the highstate.
+
+
+== Synchronize images to the Terminals
+
+.Procedure: Synchronizing images to the terminals
+
+. Power on your POS terminals.
+The branch server will bootstrap the terminals according to the data you have provided.
+. TODO:Something like this?
+. On the {susemgr} server, run this command:
++
+----
+salt <branch_server_salt_id> state.apply image-sync
+----
```

Comment:
Synchronizing images to the _branch server_ and deploying images to the terminal during boot are rather independent steps. I think this should be two sections.
The command is correct for syncing to branch server.